opi4/kernel.config

```diff
@@ -0,0 +1,5 @@
+CONFIG_TYPEC=y
+CONFIG_DRM=y
```

Comment:
Please add a comment that: `# We need DRM built-in to show recovery/fastbootd UI which is on initrd`